salt/modules/lxc.py

```diff
@@ -989,6 +1045,10 @@ def init(name,
     if users is None:
         users = []
     dusers = ['root']
+    # think to touch the default user generated by default ubuntu template which
+    # has a really unsecure password too ...
+    if __grains__['os'] in ['Ubuntu'] and 'ubuntu' not in users:
+        dusers.append('ubuntu')
```

Comment:
Thank you, i know what getent can do for me, what should i talk about it otherwise.
The ubuntu user is a default user created by the template, so it is already exists when the container is first booted.
lsbrelease wont help in the case of custom templates based on either recent ubuntu lxc templates with no user of custom tarballs with also no ubuntu users, the script will just fail for the same reason as the code i wanted to reintroduce ;-).

If there is no such user, with a getent test prior to the password set call, we can handle more gracefully and log or even filter out completly the missed password set

So, This makes my statement still valid.